backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/AggregationDAO.java

```diff
@@ -95,10 +101,28 @@ public I_C_Invoice_Candidate_Agg retrieveAggregate(final I_C_Invoice_Candidate i
 	}
 
 	@Override
-	public int findC_Invoice_Candidate_HeaderAggregationKey_ID(I_C_Invoice_Candidate ic)
+	public int findC_Invoice_Candidate_HeaderAggregationKey_ID(@NonNull final I_C_Invoice_Candidate ic)
+	{
+		try
+		{
+			return findC_Invoice_Candidate_HeaderAggregationKey_ID0(ic);
+		}
+		catch (final DBUniqueConstraintException e)
```

Comment:
hmm, using exceptions to control standard flow is not great... but we can live with it in this context